pkgs/os-specific/linux/nvme-cli/default.nix

```
@@ -11,7 +11,7 @@ stdenv.mkDerivation rec {
     sha256 = "1d538kp841bjh8h8d9q7inqz56rdcwb3m78zfx8607ddykv7wcqb";
   };
 
-  nativeBuildInputs = [ pkg-config ];
+  nativeBuildInputs = [ pkg-config libuuid ];
```

```suggestion
  nativeBuildInputs = [ pkg-config ];
  
  buildInputs = [ libuuid ];
```

otherwise it won't work when cross compiling.

---

Ahhh!!! Thanks!